pages/learn/integration/pre-built-integrations/hubspot-contacts-to-google-contacts-integration.js

```diff
@@ -230,12 +230,12 @@ export default function Learn({ samples, content }) {
               <Container>
                 <Row>
                   <Col xs={12} lg={6} style={{ fontSize: "18px" }}>
-                    <p>Synchronizing contact storages across platforms like smartphones, email, and social networks
+                    <p>Synchronizing contact storages across platforms like smart phones, emails, and social networks
```

Comment:
Smartphone is a single word.

https://en.wikipedia.org/wiki/Smartphone

Reply:
Fixed.